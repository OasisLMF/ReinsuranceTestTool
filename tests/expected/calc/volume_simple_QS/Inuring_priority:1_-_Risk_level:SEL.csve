portfolio_number,policy_number,account_number,location_number,location_group,coverage_type_id,peril_id,tiv,loss_pre,loss_net
