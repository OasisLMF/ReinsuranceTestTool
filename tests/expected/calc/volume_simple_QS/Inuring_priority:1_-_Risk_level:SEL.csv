portfolio_number,policy_number,account_number,location_number,location_group,cedant_name,producer_name,lob,country_code,reins_tag,coverage_type_id,peril_id,tiv,loss_pre,loss_net
